Frontend/ui-driver/src/Flow.purs

```diff
@@ -931,6 +932,7 @@ driverProfileFlow = do
       pure $ setText (getNewIDWithTag "VehicleRegistrationNumber") ""
       modifyScreenState $ AddVehicleDetailsScreenStateType (\_ -> defaultEpassState.addVehicleDetailsScreen)
       addVehicleDetailsflow true
+    SUBCRIPTION -> subScriptionFlow
```

Comment:
call `updateAvailableAppsAndGoToSubs`

Reply:
@Vignesh-772 There is no difference in calling `updateAvailableAppsAndGoToSubs` or `subScriptionFlow`
They are just duplicate flow